bloodytools/simulation_objects/simulation_objects.py

```diff
@@ -653,19 +655,16 @@ def simulate(self) -> bool:
             .format(self.filename)
           )
         else:
-          self.filename = str(uuid.uuid4()) + ".simc"
+          # temporary file names
+          self.uuid = str(uuid.uuid4())
+          self.filename = "{}.simc".format(self.uuid)
+          json_filename = "{}.json".format(self.uuid)
```

Comment:
Please add `self.`

Reply:
ok so in init I would add it as
self.json_filename : str = None
Correct?

Is there a reason for adding it to the class, as opposed to a local variable?

Reply:
Sounds good. Yes the reason is, that I intend to keep simulation-related files. So input file, result-json, and the processed result-json will all be available. Therefore I'd like to keep that name around already.